src/components/admin/SpaceEditForm.tsx

```diff
@@ -202,6 +211,22 @@ export const SpaceEditForm: React.FC<SpaceEditFormProps> = ({
     [relatedVenues, space.worldId, space.id]
   );
 
+  const boothTemplateOptionList = useMemo(
+    () =>
+      Object.fromEntries(
+        relatedVenues
+          .filter(
+            ({ id, worldId, template, managedBy }) =>
+              space.worldId === worldId &&
+              id !== space.id &&
+              template === VenueTemplate.meetingroom &&
+              managedBy === undefined
+          )
+          .map((venue) => [venue.id, venue])
```

Comment:
let's combine this `filter` and `map` into a `reduce` and create a separate function for the equality checks?
We'll avoid going through the array twice in this case